integrationtest/pg/powergate.go

```diff
@@ -0,0 +1,76 @@
+package pg
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+	"path"
+	"runtime"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	pc "github.com/textileio/powergate/api/client"
+	"github.com/textileio/powergate/health"
+	"github.com/textileio/textile/util"
+	"google.golang.org/grpc"
+)
+
+var powAddr = "127.0.0.1:5002"
+
+func StartPowergate(t util.TestingTWithCleanup) *pc.Client {
+	_, currentFilePath, _, _ := runtime.Caller(0)
+	dirpath := path.Dir(currentFilePath)
```

Comment:
Little trick to make sure the `docker-compose.yml` file is found when this func is used from elsewhere in the project.